pkg/dbmate/db.go

```diff
@@ -218,6 +218,41 @@ func (db *DB) DumpSchema() error {
 	return os.WriteFile(db.SchemaFile, schema, 0o644)
 }
 
+// LoadSchema loads schema file to the current database
+func (db *DB) LoadSchema() error {
+	drv, err := db.Driver()
+	if err != nil {
+		return err
+	}
+
+	sqlDB, err := drv.Open()
+	if err != nil {
+		return err
+	}
+	defer dbutil.MustClose(sqlDB)
+
+	_, err = os.Stat(db.SchemaFile)
+	if err != nil {
+		return nil
```

Comment:
shouldn't this return a wrapped error?

Reply:
Agree: if we fail to `os.Stat()` the file, we should raise an error and not terminate successfully.